src/modm/communication/amnb/handler.hpp.in

```diff
@@ -342,8 +342,11 @@ protected:
 %% if with_io
 #include <modm/io/iostream.hpp>
 
+namespace modm
+{
+
 inline modm::IOStream&
-operator << (modm::IOStream& s, const modm::amnb::Error error)
+operator << (modm::IOStream& s, const modm::amnb::Error& error)
```

Comment:
`modm::amnb::Error` is a `uint8_t` enum, I would pass it by value, not by const ref.